### PR TITLE
Add test dependency for pubsub

### PR DESCRIPTION
**Pull request by shinfan**

### CODE REVIEW

File: gapic/packaging/dependencies.yaml [hunk 1]
```diff
@@ -49,7 +49,11 @@ gax_version:
   ruby:
     lower: '0.8.0'
   java:
-    lower: '0.6.1'
+    lower: '1.2.0'
+
+gax_grpc_version:
```
Comment (garrettjonesgoogle) [inline gapic/packaging/dependencies.yaml]: Do you have to make a code change to use this?

Comment (shinfan) [inline gapic/packaging/dependencies.yaml]: Yes, I have one PR pending that will use this. This should not break the existing code though. I can revert this for now for safety.

**Comment by shinfan** [comment]
@garrettjonesgoogle PTAL

File: gapic/packaging/dependencies.yaml (hunk 1 above)
Comment (garrettjonesgoogle) [inline gapic/packaging/dependencies.yaml]: Hmm well gax 1.2.0 is incomplete, so someone using only this shouldn't be able to build a coherent package, right?

Comment (shinfan) [inline gapic/packaging/dependencies.yaml]: ahhh okay I have reverted the whole file and add the change later once my code PR is complete.

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**